docs/operations/metrics.md

```diff
@@ -323,6 +323,8 @@ decisions.
 |`segment/usedPercent`|Percentage of space used by served segments.|dataSource, tier, priority.|< 100%|
 |`segment/count`|Number of served segments.|dataSource, tier, priority.|Varies.|
 |`segment/pendingDelete`|On-disk size in bytes of segments that are waiting to be cleared out|Varies.|
+| `segment/averageRowCount`       | The average number of rows per segment in a datasource               | dataSource, tier, priority.                 | ~5 million |
+| `segment/numOfSegmentsInBucket` | The number of segments in a bucket.                                  | dataSource, tier, priority, rowCountBucket. |Varies.|
```

Comment:
```suggestion
| `segment/rowCount/range/count` | The number of segments in a bucket.                                  | dataSource, tier, priority, range. |Varies.|
```
how about `range` or `bucket` to indicate the buckets of the distribution. I also thought of using the word `distribution` in the metric name

Reply:
This and the one above seem like they should have similar scheme for these.
`segment/rowCount/range/count `
`segment/rows/avg` so im thinking about using this instead `segment/rowCount/avg`